docs/rms_event_design.md

```diff
@@ -1 +1,52 @@
+
+The purpose of this package is to provide practical guidance and a core set of metadata to implement rapid mortality surveillance in national DHIS2-based health information systems. This package is aligned to the [WHO's Technical Package for Rapid Mortality Surveillance and Epidemic Response](https://www.who.int/publications/i/item/revealing-the-toll-of-covid-19) (2020) for assessing the toll of COVID-19 in countries.  It is designed to be flexible and feasible for use in low-resource settings, including contexts where data collection largely remains paper-based.
+
+Reporting of total mortality from facilities, communities and midolegal personnel (as appropriate to country context) with basic data such as age, sex, date of death, place of death and place of residence enables a more complete picture of impact, particularly for deaths that may occur in the home/the community or may be indirectly linked to COVID-19. In addition, analyses of year-over year trends pre- and post-epidemic can reveal indirect impact related to disruptions in access to health services and products. 
+
+## 2. System Design Overview
+
+### 2.1 Use case summary
+
+According to the WHO's technical package, RMS requires 1) a source of rapidly and routinely reported deaths by age, sex and location; and 2) some means to establish a baseline of pre-epidemic mortality levels by age and sex against which to compare the current reports. The metadata package corresponds to these components with an event program for line-listed data collection of deaths optimized for facility and community reporting; and a reference dashboard that enables year-over-year trend analysis to examine excess deaths compared to pre-pandemic levels. 
+
+Unlike cause-based mortality surveillance systems and death certification systems, the intent of RMS is to capture *total* mortality for the purpose of epidemic impact monitoring. Data are intended to be collected both in the facility and in the community, to capture all deaths. This approach can overcome known shortfalls to achieving completeness and coverage of cause-based mortality systems in many countries, such as poor linkages between health facility death reporting and national CRVS systems as well as high proportions of deaths that occur in the community and may go unreported. 
+
+For cause-based mortality surveillance, please reference the [DHIS2 Cause of Death metadata package](https://docs.dhis2.org/en/topics/metadata/crvs/cause-of-death-tracker-and-events/design.html) designed to support the WHO recommended medical certification of cause of death and International Classification of Disease (ICD) standards.
```

Comment:
Here, it would be enough to use unique reference instead of link:

[DHIS2 Cause of Death metadata package](#cod-design)